crates/cli/src/build.rs

```diff
@@ -242,6 +244,7 @@ pub fn build_file<'a>(
     let mut loaded = loaded;
     let problems = program::report_problems_monomorphized(&mut loaded);
     let expectations = std::mem::take(&mut loaded.expectations);
+    let layout_interner = loaded.layout_interner.clone();
```

Comment:
bit sad but I don't see a way around it

Reply:
Can we return the interner from  `program::gen_from_mono_module` instead of doing this?

Reply:
the problem is we might end up inserting stuff that then is not present in the interner, which could be a problem

Reply:
this worked out well, we also don't need to clone some other stuff now